cmd/cluster-kube-scheduler-operator/render/render.go

```diff
@@ -103,7 +113,21 @@ func (r *renderOpts) Run() error {
 		return err
 	}
 
-	return genericrender.WriteFiles(&r.generic, &renderConfig.FileConfig, renderConfig)
+	// TODO: remove after the transition in the installer to a phase-2 free bootstrapping
+	var filters []assets.FileInfoPredicate
+	if r.disablePhase2 {
+		filters = append(filters, func(info os.FileInfo) bool {
+			if strings.HasPrefix(info.Name(), "kube-system-") {
+				return false
+			}
+			if info.Name() == "daemonset-kube-scheduler-daemonset.yaml" {
```

Comment:
i think name of the file here should match to `kube-scheduler-daemonset.yaml` as here https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/bindata/bootkube/manifests/kube-scheduler-daemonset.yaml ?

Reply:
indeed, typo

Reply:
fixed